endpoints/lib/vboxServiceWrappers.php

```diff
@@ -7881,6 +7881,10 @@ public function resize($arg_logicalSize)
 
     public function reset()
     {
+        // No password ID in case of decryption and de-/encryption are both handled here.
```

Comment:
Sorry to be a pedant, but I'm curious about these 4 lines.  They aren't simply recasting like the rest of the patch.  These lines already exist in changeEncryption and you're not blanking the vars like I'd expect in reset().  Can you explain this further?

Reply:
It is from https://github.com/phpvirtualbox/phpvirtualbox/pull/55/commits/ab7059433b7c7f7232d3e33435f02e920a9a2702
I merge together my changes with changes from other authors.
I will remove it.

Reply:
Yeah, that patch is patching changeEncryption(), which we have already done.  Whereas your patch add these lines to reset().  See above.